lib/commands/general.js

```diff
@@ -234,6 +235,10 @@ commands.mobileViewportScreenshot = async function () {
   return await this.getViewportScreenshot();
 };
 
+commands.mobileViewPortRect = async function () {
```

Comment:
Please add a docstring on the method about what it is supposed to return

Reply:
I copied `mobileViewportScreenshot` implementation in this file to `mobileViewPortRect`, which isn't documented with the return value, and I didn't see any other method documented with return value. The actual `commands.getViewPortRect` method was already implemented in the `viewport.js` file. So I don't understand what should I do.

Reply:
```
async function () { -> async function mobileViewPortRect () { 
```

Reply:
Here is an example: https://github.com/appium/appium-uiautomator2-driver/pull/404/files#diff-046460f6a24bb8c575cc6d5f55189736R217

Reply:
You mean like in the last method in that file? `mobileType`? This is the only method implemented that way...

Reply:
You mean like in the last method in that file? `mobileType`? This is the only method implemented that way...

Reply:
yes, similar to that one

Reply:
Done in a recent commit https://github.com/appium/appium-uiautomator2-driver/pull/404/commits/83dfefed16382e993d31b26f4efd6ccf1eecee42 from yesterday.

Reply:
I hope commit https://github.com/appium/appium-uiautomator2-driver/pull/404/commits/63d0703948b9bf5913f0c354db50018cb84103b5 was done correctly.